__tests__/websocket/websocket.test.ts

```diff
@@ -46,9 +46,11 @@ describe('onMessage', () => {
 
   it('should receive messages after initializing the timeoutTimer', () => {
     const wsInstance = new GenericWebSocket(baseWsOptions);
+    const msgSpy = jest.spyOn(wsInstance, 'onMessage').mockImplementation(jest.fn());
 
     wsInstance.sendPing();
     wsInstance.onMessage({ data: JSON.stringify({ type: 'other' }) });
+    expect(msgSpy).toHaveBeenCalled();
```

Comment:
So this test wasn't doing anything?

Reply:
That's correct. It would throw if the code failed, so it increased our coverage, but had no assertions.